packages/server/src/db/index.js

```diff
@@ -203,7 +201,6 @@ async function getUser(id) {
             id: user.tenant_id,
             display_name: user.tenant_display_name,
             main_agency_id: user.tenant_main_agency_id,
-            uses_spoc_process: user.tenant_uses_spoc_process,
```

Comment:
I searched the entire project for the string "uses_spoc_process" and found matches only in migration files.